README.md

```diff
@@ -146,44 +144,9 @@ Note: Podman is CLI-compatible with Docker, so simply replace "docker" with "pod
 ## Enable OCR/Compression feature
 Please view https://github.com/Stirling-Tools/Stirling-PDF/blob/main/HowToUseOCR.md
 
-## Want to add your own language?
-Stirling PDF currently supports 26!
```

Comment:
Although removing it from the how to add makes sense i do like listing the supporting languages on the main readme for people

Reply:
Sure, we can keep it in the readme.